lib/claim_letters/claim_letter_downloader.rb

```diff
@@ -41,7 +32,7 @@ def get_letter(document_id)
 
       filename = filename_with_date(letter_details[:received_at])
 
-      if !Rails.env.development? && !Rails.env.test?
+      if !Rails.env.development? && !Rails.env.test? && !Settings.vsp_environment == 'development'
```

Comment:
Should this be `Settings.vsp_environment != 'development'`?

Reply:
doh! good catch